app/uk/gov/hmrc/tariffclassificationfrontend/models/SampleStatus.scala

```diff
@@ -18,11 +18,12 @@ package uk.gov.hmrc.tariffclassificationfrontend.models
 
 object SampleStatus extends Enumeration {
   type SampleStatus = Value
-  val AWAITING, MOVED_TO_ACT, MOVED_TO_ELM, SENT_FOR_ANALYSIS, SENT_TO_APPEALS, STORAGE, RETURNED_APPLICANT,
+  val NONE, AWAITING, MOVED_TO_ACT, MOVED_TO_ELM, SENT_FOR_ANALYSIS, SENT_TO_APPEALS, STORAGE, RETURNED_APPLICANT,
```

Comment:
What does this give us having an option called NONE, over just setting it to an Option value of None? Far as I can see everywhere it's used in a model object it's optional.

Reply:
It's needed because we have chosen to re-purpose `None` as meaning "liability sample not being sent".  Now `Some(NONE)` means "no sample status and Some(_) means "liability sample is being sent (for a liability case).

Hope that makes sense.

Reply:
Ahh OK got you - couldn't get that from my brief look, but fair enough!